Source/Delphi.Mocks.Helpers.pas

```diff
@@ -33,9 +33,20 @@
 interface
 
 uses
-  Rtti;
+  System.Generics.Collections,
+  System.Rtti;
 
 type
+  //Allow custom comparisons
+  TCustomValueComparer = reference to function(const a, b: TValue): Integer;
```

Comment:
No idea if it's possible, but did you try doing something like 
````TCustomValueComparer = reference to function<T>(const a, b: T): Integer;````
that would allow us to avoid TValue

Reply:
I don't think that would be possible as I think you might need to declare it as `TCustomValueComparer<T> = reference to function<T>(const a, b: T): Integer;` so it couldn't be stored in a common container. Having said that, I am sure there should be a way to do this without the need for TValue?